### DISCUSSION TITLE
BuildSpeed или BuildFast

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#92] [Ideas]
Добавить свойство в Api, с ограниченным значением, для того чтобы установить скорость стройки игрока.

**Comment by justqstn** [comment]
не имеет особого смысла, сейчас и так все нормально

**Comment by kkohno** [comment]
ну тут можно сделать конечно, только с ограничениями от и до... однако есть еще балки у нас, которые стройку ускоряют

**Comment by KukushkaSex** [comment]
Ок
